bundles/core-combat/player-events.js

```diff
@@ -17,12 +17,12 @@ module.exports = (srcPath) => {
         // could potentially wind up in a situation where the player performed
         // a mid-round attack that killed the target, then the next round the
         // target kills the player. So let's not let that happen.
-        if (this.attributes.health <= 0) {
+        if (this.getAttribute('health') <= 0) {
           this.combatants.forEach(combatant => {
             this.removeCombatant(combatant);
             combatant.removeCombatant(this);
             if (!combatant.isInCombat()) {
-              combatant.attributes.health = combatant.attributes.maxHealth;
+              combatant.setAttribute('health', combatant.getRawAttribute('maxHealth'));
```

Comment:
This should probably be `getAttribute` instead of `getRawAttribute` in, for example, there was an effect on the Npc that increased or lowered its max health